wolfssl/error-ssl.h

```diff
@@ -102,7 +102,7 @@ enum wolfSSL_ErrorCodes {
     COOKIE_ERROR                 = -369,   /* dtls cookie error */
     SEQUENCE_ERROR               = -370,   /* dtls sequence error */
     SUITES_ERROR                 = -371,   /* suites pointer error */
-
+    MAX_CERT_EXTENSIONS_ERR      = -372,   /* max cert extension exceeded */
```

Comment:
Fails error tests for me:

`./configure --enable-all --enable-ocspstapling=no-multi`

```
./scripts/unit.test -833
starting unit tests...
 Begin API Tests
   833: test_wolfSSL_ERR_strings                            :
ERROR - tests/api.c line 83240 failed with:
    expected: strcmp((errStr),(unknownStr)) == 0
    result:   -40 != 0


ERROR - tests/api.c line 83286 failed with:
    expected: error_test() == 1
    result:   -372 != 1

 failed (  0.00006)

ERROR - tests/api.c line 91637 failed with:
    expected: Test failed

    result:   ret 0


FAILURES:
   833: test_wolfSSL_ERR_strings

 End API Tests
```

Reply:
Hi @JacobBarthelmeh 
I cannot see the error. Is there any pre-setting needed for testing? I checked the following gcc and clang version:
```
$ gcc --version
gcc (Ubuntu 9.5.0-1ubuntu1~22.04) 9.5.0
$ gcc --version
gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
$ clang --version
Ubuntu clang version 14.0.0-1ubuntu1.1
$ lsb_release -d
Description:    Ubuntu 22.04.5 LTS
```

Used command:
```
./configure --enable-all --enable-ocspstapling=no-multi && make check
```
Could you share your test environment?

Reply:
After rebasing to the latest master, I am seeing the errors now.